homeassistant/components/zha/core/const.py

```diff
@@ -157,20 +157,23 @@ class RadioType(enum.Enum):
     """Possible options for radio type."""
 
     ezsp = (
-        "ESZP: HUSBZB-1, Elelabs, Telegesis, Silabs EmberZNet protocol",
+        "EZSP = Silicon Labs EmberZNet protocol (ex; Elelabs, HUSBZB-1, Telegesis)",
```

Comment:
IMO should be a colon in `ex:` since the models are closely related to the previous clause

Reply:
OK done